SPECS/rpm/rpm.spec

```diff
@@ -274,8 +272,14 @@ popd
 %files -n python3-rpm
 %defattr(-,root,root)
 %{python3_sitelib}/*
+%{_libdir}/rpm/pythondistdeps.py
+%{_fileattrsdir}/python.attr
```

Comment:
When I look at the `python3-rpm` package in Fedora, I think they only include the parts inside the sitelib directory as well. In addition to that the `pythondistdeps.py` and `python.attr` contents look like something, that might be useful for building RPM packages. Considering that, I think it might be better to place these two inside `rpm-build`. At least all of the `*.attr` files (including `python.attr` - we have it packaged twice right now) seem to be already and the macros inside `python.attr` require `pythondistdeps.py`.